Dockerfile

```diff
@@ -1,24 +1,40 @@
-FROM node:lts-alpine
-ENV NPM_CONFIG_LOGLEVEL warn
+FROM node:16.15.0-alpine
+
+ARG VERSION_INFO=noVersion
 
-# Install "git"
-RUN apk update \
- && apk add alpine-sdk
```

Comment:
This was installing `git` and a bunch of other stuff into the image, and the only reason I could find was the `git describe --tags` that gets run in `config/version.js` to grab the latest commit hash/tag to display on the lobby page. Installing all that stuff was slow and unnecessary just to show the latest commit/tag on that page. Now the `git describe --tag` call happens outside the build and gets passed in as an ARG, which sets the environment variable `VERSION_INFO` later on in the build, hence the refactor in `config/version.js`. This cuts down on the time needed to build the image and cuts down on the size of the image somewhat as well.